src/operator-docs/transformation/bufferToggle.ts

```diff
@@ -1,6 +1,94 @@
 import { OperatorDoc } from '../operator.model';
 
 export const bufferToggle: OperatorDoc = {
-  'name': 'bufferToggle',
-  'operatorType': 'transformation'
+  name: 'bufferToggle',
+  operatorType: 'transformation',
+  signature: `bufferToggle(
+    openings: SubscribableOrPromise<O>,
+    closingSelector: (value: O) => SubscribableOrPromise): Observable<T[]>`,
+  parameters: [
+    {
+      name: 'openings',
+      type: 'SubscribableOrPromise<O>',
+      attribute: '',
+      description: `A Subscribable or Promise of notifications to start new buffers.`
+    },
+    {
+      name: 'closingSelector',
+      type: '(value: O) => SubscribableOrPromise',
+      attribute: '',
+      description: `A function that takes the value emitted by the openings observable
+      and returns a Subscribable or Promise, which, when it emits, signals that the associated buffer should be emitted and cleared.`
+    }
+  ],
+  marbleUrl: 'http://reactivex.io/rxjs/img/bufferToggle.png',
+  shortDescription: {
+    description: `
+    Buffers the source Observable values starting from an emission from <span class="markdown-code">openings</span>
+    and ending when the output of <span class="markdown-code">closingSelector</span> emits.
+    <span class="informal">
+    Collects values from the past as an array. Starts collecting only when <span class="markdown-code">openings</span> emits,
+    and calls the <span class="markdown-code">closingSelector</span> function
+    to get an Observable that tells when to close the buffer.</span>`
+  },
+  walkthrough: {
+    description: `
+    Buffers values from the source by opening the buffer via signals from an Observable
+    provided to <span class="markdown-code">openings</span>,
+    and closing and sending the buffers when a Subscribable or Promise
+    returned by the <span class="markdown-code">closingSelector</span> function emits.
+    `
+  },
+  examples: [
+    {
+      name: 'Every other second, emit the click events from the next 500ms',
+      code: `
+      import { fromEvent } from 'rxjs/observable/fromEvent';
+      import { interval } from 'rxjs/observable/interval';
+      import { empty } from 'rxjs/observable/empty';
+      import { map, bufferToggle } from 'rxjs/operators';
+
+      const clicks$ = fromEvent(document, 'click');
+      const openings$ = interval(1000);
+      const buffered$ = clicks$.pipe(
+        map(e => {return {X: e.clientX, Y: e.clientY};}),
+        bufferToggle(openings$, i => i % 2 ? interval(500) : empty())
+      );
+      buffered$.subscribe(x => console.log(x));
```

Comment:
Please add expected output for the examples also.

Reply:
@sumitarora  I don't really understand what you mean... Should I add a comment that explains what you'll see when you run this code? Or should I take some output from when I run it and put that in comment together with this code?

Reply:
@DiedrikDM See the output block as part of code explaining expected output. https://github.com/ReactiveX/rxjs-docs/blob/master/src/operator-docs/combination/combineLatest.ts#L62

Reply:
@sumitarora ok, I'll do that :) 👍